### DISCUSSION TITLE
Compatibility with ForcePack

### DISCUSSION THREAD

**Discussion by neomoth** [spartacus04/jext-reborn#314] [Ideas]
Would like to know if it would be possible to allow for compatibility with ForcePack, a plugin to make updating the server resource pack a lot easier and not require a server restart, as well as allowing for sending multiple resource pack files to clients if the server version is running on 1.20.4, making so the file size limit is less of an immediate issue
Pack would need to be capable of splitting itself once going over 250mb, and have all parts able to be detected and served by ForcePack. Would this be feasible?

**Comment by spartacus04** [comment]
That could be an awesome improvement, however I don't really know how I could implement resourcepack slicing. Some user reported a bug a few days ago where he was importing hundreds of discs, and that would go over the 250mb limit. Also I would need to add compatibility for lower versions where the limit is 100/50mb. Overall I could implement the slicing feature in the webui but that wouldn't be really optimal. I'll try to find a solution for this because this could be a really good idea.
